contracts/src/v0.8/functions/dev/0_0_0/FunctionsBillingRegistry.sol

```diff
@@ -67,6 +67,8 @@ contract FunctionsBillingRegistry is
   }
   // Note a nonce of 0 indicates an the consumer is not assigned to that subscription.
   mapping(address => mapping(uint64 => uint64)) /* consumer */ /* subscriptionId */ /* nonce */ private s_consumers;
+  mapping(address => uint64) /* consumer */ /* consumer_id */ private s_consumerIds;
+  uint64 private lastConsumerId;
```

Comment:
Should not this be "consumer nonce"?

Reply:
Technically yes, however s_consumers is already holding a nonce (different one) so it'd be a little confusing.